LandSandBoat/server
# [sql/lua] Dyinyinga NM

**_I affirm:_**

## What does this pull request do?

~~LUA side of [this pr](https://github.com/LandSandBoat/server/pull/5510)~~

[Dyinyinga](https://www.bg-wiki.com/ffxi/Dyinyinga) has a neat aura mechanic when the AE isn't applied to its target

https://imgur.com/1o5UQVS

## Steps to test these changes

Update the slug's lua to spawn the NM immediately on despawn: `xi.mob.phOnDespawn(mob, ID.mob.DYINYINGA_PH, 100, 3600, { immediate = true }) -- 1 hour`

Kill the PH

Fight NM, see it does the things

kill and see that it has drops
